docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2188` Users can now specify the exact second a ledger action happened when adding or editing ledger actions.
```

Comment:
When you also make it work for trades (and any other events) rephrase to:

Users can now specify the datetime for events (ledger actions, trades) to seconds precision.